services/riak/values.yaml

```
@@ -53,11 +53,10 @@ tolerations: []
 
 affinity: {}
 
-config: 
+config:
   user: {}
```

```suggestion
  user: ""
```

Вроде тут теперь строка ожидается